src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import Menu from './Menu'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+
+class AccountMenuDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
@jrob which line? I was thinking you wanted space between lines 5 and 6? or between line 6 and 7?